kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/KafkaStreamsApp.java

```diff
@@ -140,6 +140,18 @@ protected void doStop() {
     app.close(Duration.ofSeconds(30));
   }
 
+  /**
+   * This method is invoked just before a subtopology is created
+   * Any dependencies that need to be initialized need to be done here
+   * @param subTopologyJobConfig
+   */
+  protected void doInitForConsolidatedDeployment(Config subTopologyJobConfig){}
```

Comment:
nit: `doInitForConsolidatedDeployment` - `doInitForConsolidatedKStreamsApp` or ``doInitForConsolidatedService` ?

Reply:
made the change